packages/shared-lib/src/app/types/easy-genomics/laboratory-run.d.ts

```diff
@@ -38,4 +38,8 @@ export interface LaboratoryRun extends BaseAttributes {
   S3Input?: string;
   S3Output?: string;
   Settings?: string; // JSON
+  CreatedAt?: string;
+  CreatedBy?: string;
+  ModifiedAt?: string;
+  ModifiedBy?: string;
```

Comment:
These additional properties for the LaboratoryRun table are not needed here, as it already extends the BaseAttributes which define these properties.